app/controllers/users_controller.rb

```diff
@@ -16,10 +16,24 @@ def destroy
     redirect_to users_url
   end
   
+  def edit_info
+    @user = User.find(params[:id])
+  end
+  
   def edit
```

Comment:
`edit` is a little ambiguous now.  Maybe we should rename it to `edit_password`.